sir_cake/seller/utils.py

```diff
@@ -1,10 +1,34 @@
-def crop_center(img, cropped_width, cropped_height):
-    img_width, img_height = img.size
-    return img.crop(((img_width - cropped_width) // 2,
-                    (img_height - cropped_height) // 2,
-                    (img_width + cropped_width) // 2,
-                    (img_height + cropped_height) // 2))
+from django.core.paginator import Paginator
 
+from .models import Item
 
-def crop_max_square(img):
-    return crop_center(img, min(img.size), min(img.size))
+
+def all_products_context(request):
```

Comment:
Why is this function in the utils file? Will you need to call that same function from other views? Otherwise it is fine if you simply move this logic directly to the `all_items` view.

Reply:
This function is un utils.py, because it is going to be used in another view